relay-config/src/config.rs

```diff
@@ -366,6 +372,8 @@ impl Default for Relay {
             tls_port: None,
             tls_identity_path: None,
             tls_identity_password: None,
+            upstream_reauthentication_interval: 60,
+            upstream_reauthentication_grace_period: 20,
```

Comment:
After looking at this, I'd suggest to move to `http` and rename to `http.auth_interval` and `http.auth_grace_period`. We also define connection timeouts, retry intervals and similar there, so it seems like a good fit.

Reply:
ok, will do